mmaction/datasets/base.py

```diff
@@ -37,6 +38,14 @@ class BaseDataset(Dataset, metaclass=ABCMeta):
             from 0. Default: 1.
         modality (str): Modality of data. Support 'RGB', 'Flow'.
             Default: 'RGB'.
+        sample_by_class (bool): Sampling by class, should be set `True` when
+            you want to perform inter-class data balancing. Only compatible
+            with `multi_class == False`. Only applies for training. Default:
+            False.
+        power (float): We support sampling data with the probability
+            proportional to the power of its label frequency (freq ^ power).
```

Comment:
`probability proportional to the power of its label frequency (freq ^ power) when sampling data.`